go-controller/pkg/util/kube.go

```diff
@@ -716,7 +716,7 @@ func ExternalIDsForObject(obj K8sObject) map[string]string {
 	if gk.String() == "" {
 		kinds, _, err := scheme.Scheme.ObjectKinds(obj)
 		if err != nil || len(kinds) == 0 || len(kinds) > 1 {
-			klog.Warningf("BUG: object has no / ambiguous GVK: %#v, err", obj, err)
+			klog.Warningf("Object %v has no ambiguous GVK: %#v, err", obj, err)
```

Comment:
`Object %v has no or ambiguous GVK` ...

Reply:
Since we're being this picky, wouldn't it be better to say `Object %v doesn't have or has ambiguous GVK` ? ;)

Reply:
yeah I changed this to: `klog.Warningf("Object %v either has no GroupVersionKind or has an ambiguous GroupVersionKind: %#v, err", obj, err)`